apps/public-docsite-v9/.storybook/preview.js

```diff
@@ -59,3 +59,33 @@ export const parameters = {
     },
   },
 };
+
+const slotRegex = /as\?:\s*"([^"]+)"/;
+
+/**
+ *
+ * @type {import('@storybook/types').ArgTypesEnhancer}
+ */
+const withSlotEnhancer = context => {
```

Comment:
this works as expected with default DocsPage as it updates context properly which is then obtained and used for acutall rendering.

https://github.com/storybookjs/storybook/blob/release-7-6/code/ui/blocks/src/blocks/ArgsTable.tsx#L220


because we uverride DocsPage this is no longer working unfortunately, as the context won't propagate as expected within SB ArgsTable component.

_Task:_
- [ ] determine if this can be set properly so SB ArgsTable would render enhanced data